src/client/components/search/index.test.js

```diff
@@ -29,11 +29,15 @@ describe('mapStateToProps', () => {
     selectors.searchCurrentTextSelector = jest.fn()
       .mockReturnValue('MSFT');
 
+    selectors.searchErrorSelector = jest.fn()
+      .mockReturnValue(null);
+
     selectors.stocksSelector = jest.fn()
       .mockReturnValue(mockStocks);
 
     expect(mapStateToProps(mockState)).to.deep.equal({
       text: 'MSFT',
+      hasError: false,
       stocks: mockStocks
```

Comment:
consider

`const mockText = 'MSFT'`

and use it in the two places it says 'MSFT'